_data/translations.yml

```diff
@@ -172,8 +172,8 @@ header-language-picker-current-lang:
 #   en: 
   
 govlogo:
-  nl: Home
-  en: Home
+  nl: Logo Rijksoverheid, homepage
+  en: Logo Dutch Government, homepage
```

Comment:
Don't forget to add the other translations.

Reply:
I've put in a request for them with our content editor! My idea was to add them when I get them.